gulpfile.js

```diff
@@ -126,7 +126,7 @@ gulp.task('validate', function() {
  * Watch for changes and recompile
  */
 gulp.task('watch', function() {
-  return watch(['./src/**/*.{js,less,handlebars}', './swagger-spec/**/*.yaml'], function() {
+  return watch(['./src/**/*.{js,less,handlebars}'], function() {
```

Comment:
You actually don't need to revert this change here. You can pull the develop branch into this branch to resolve this.